precompile/contracts/txallowlist/module.go

```diff
@@ -0,0 +1,47 @@
+// (c) 2019-2020, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package txallowlist
+
+import (
+	"fmt"
+
+	"github.com/ava-labs/subnet-evm/precompile/config"
+	"github.com/ava-labs/subnet-evm/precompile/contract"
+	"github.com/ava-labs/subnet-evm/precompile/modules"
+	"github.com/ethereum/go-ethereum/common"
+)
+
+// ConfigKey is the key used in json config files to specify this precompile config.
+// must be unique across all precompiles.
+const ConfigKey = "txAllowListConfig"
+
+var ContractAddress = common.HexToAddress("0x0200000000000000000000000000000000000002")
+
+var Module = modules.Module{
+	ConfigKey:    ConfigKey,
+	Address:      ContractAddress,
+	Contract:     TxAllowListPrecompile,
+	Configurator: &configuror{},
+}
+
+type configuror struct{}
```

Comment:
can we change this to be `configurator` to match the interface name and add a declaration at the top to enforce a type check within this file that it matches the interface?

Reply:
Same comment for the rest of the modules

Reply:
done